<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorToolbarEditor</class>
 <widget class="QWidget" name="ColorToolbarEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="whatsThis">
      <string>Here you can alter the layout of the quick colors toolbar. Use the controls to add, remove or edit the colors used. You can also add separators in between colors.

Usually this contains MainColor, EdgeColor and some auxiliary colors used to group objects.</string>
     </property>
     <property name="title">
      <string>Color Toolbar</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QListView" name="colorToolbarView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="quickColor_add">
          <property name="text">
           <string>Add Color</string>
          </property>
          <property name="icon">
           <iconset resource="../../ldforge.qrc">
            <normaloff>:/icons/palette.png</normaloff>:/icons/palette.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quickColor_addSep">
          <property name="text">
           <string>Add Separator</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quickColor_edit">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../ldforge.qrc">
            <normaloff>:/icons/mode-draw.png</normaloff>:/icons/mode-draw.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quickColor_moveUp">
          <property name="text">
           <string>Move Up</string>
          </property>
          <property name="icon">
           <iconset resource="../../ldforge.qrc">
            <normaloff>:/icons/arrow-up.png</normaloff>:/icons/arrow-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quickColor_moveDown">
          <property name="text">
           <string>Move Down</string>
          </property>
          <property name="icon">
           <iconset resource="../../ldforge.qrc">
            <normaloff>:/icons/arrow-down.png</normaloff>:/icons/arrow-down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quickColor_remove">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../ldforge.qrc">
            <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quickColor_clear">
          <property name="text">
           <string>Clear List</string>
          </property>
          <property name="icon">
           <iconset resource="../../ldforge.qrc">
            <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ldforge.qrc"/>
  <include location="../../ldforge.qrc"/>
 </resources>
 <connections/>
</ui>
